suite-common/connect-init/src/actions/prepareConnectInit.ts

```diff
@@ -0,0 +1,70 @@
+import {
```

Comment:
Can we rename this file `slice.ts` and move it to root, to follow conventions?

Reply:
https://github.com/trezor/trezor-suite/pull/5760/commits/979f44f2e30665c851548bc75ae31ae9d9bbba9b